compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -1183,7 +1183,11 @@ impl<'ra, 'tcx> Resolver<'ra, 'tcx> {
             let in_module_is_extern = !in_module.def_id().is_local();
             in_module.for_each_child(self, |this, ident, ns, name_binding| {
                 // avoid non-importable candidates
-                if !name_binding.is_importable() {
+                if !name_binding.is_importable()
```

Comment:
@oli-obk Would it be better to reject assoc const or fn here? It emits suggestions to import those associated items from traits.

Reply:
The disadvantage is: people have to add similar codes to filter out assoc const or fn where the missing feature leads to consufing behaviours.

Reply:
Yea this change seems much better to me than what I said, thanks!